### ISSUE TITLE
[Adaptive Cards-InputsWithValidation.json]: Screen reader is not announcing the position information for the radio controls present below the 'Input.ChoiceSet' heading text on the 'InputsWithValidation.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9232]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputsWithValidation.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputsWithValidation.json' adaptive card will appear on right sight of the pane.
4. Navigate to the radio controls present below the 'Input.ChoiceSet' heading text using tab key.
5. Verify that, screen reader is announcing the position information for the radio controls present below the 'Input.ChoiceSet' heading text on the 'InputsWithValidation.json' card or not.

**Actual Result:**
Screen reader is not announcing the position information (e.g., '1 of 3') for the radio controls present below the 'Input.ChoiceSet' heading text on the 'InputsWithValidation.json' card.

**Observation:**
Same issue replicates with NVDA.

**Expected Result:**
The screen reader should announce the position information (e.g., "1 of 3", "2 of 3") for each radio control present under the 'Input.ChoiceSet' heading on the 'InputsWithValidation.json' card. This allows users to understand their relative position within the group of choices while navigating.

**User Impact:**
Visually impaired users who rely on screen readers will be impacted if screen reader not announcing the position information on navigating to the radio controls, the lack of position information prevents them from understanding how many options are available and which one is currently focused. This makes it harder to make accurate selections and reduces confidence while interacting with the form.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/ff506d46-879f-447b-ace9-f7044ae07966)

https://github.com/user-attachments/assets/a0d6d39c-ccc0-44a1-871e-8e6f1ebf289c

### Card JSON

```JSON
N/A
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
